src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,79 @@
+package lotto.domain;
+
+import lotto.exception.ErrorMessage;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+
+// 로또
+public class Lotto {
+
+    private List<Integer> numbers;
+    private int winningNumberMatchCount;
+
+    public Lotto(List<Integer> lottoNumbers) {
+        numberCountValid(lottoNumbers);
+        numberDuplicateValid(lottoNumbers);
+        numberRangeValid(lottoNumbers);
+        this.numbers = lottoNumbers;
```

Comment:
Collection에서 제공하는 unmodifiableList를 활용해보는 건 어떨까요?
리스트에 대한 추가, 삭제를 막아주는 기능으로 일급 컬렉션도 좀 더 불변에 가까워질 수 있습니다 :)
```suggestion
        this.numbers = Collections.unmodifiableList(lottoNumbers);
```